crates/pixi_build_types/src/procedures/conda_outputs.rs

```diff
@@ -0,0 +1,214 @@
+//! Describes the `conda/outputs` request and its parameters.
+//!
+//! This request is used to compute all the outputs that a particular backend
+//! can provide. It returns the identifiable metadata of the outputs, including
+//! the dependencies required to be able to build them.
+//!
+//! This API was introduced in Pixi Build API version 1.
+
+use ordermap::OrderSet;
+use rattler_conda_types::{NoArchType, PackageName, Platform, VersionWithSource};
+use serde::{Deserialize, Serialize};
+use serde_with::serde_as;
+use std::{
+    collections::{BTreeSet, HashMap},
+    path::PathBuf,
+};
+
+use crate::{BinaryPackageSpecV1, PackageSpecV1, project_model::NamedSpecV1};
+
+pub const METHOD_NAME: &str = "conda/outputs";
+
+/// Parameters for the `conda/outputs` request.
+///
+/// The result of this request should be a list of packages that can be built by
+/// this particular backend.
```

Comment:
```suggestion
/// this particular source dependency.
```

I guess both are correct, but it makes it a bit clearer for me.